src/DssSpell.t.sol

```diff
@@ -2200,6 +2200,150 @@ contract DssSpellTest is DSTest, DSMath {
         checkAuth(true);
     }
 
+    function check_UNIV2_replacement(bytes32 ilk, LPOsmAbstract lpPip, LPOsmAbstract oldLpPip) internal {
+        try MedianAbstract(oldLpPip.orb0()).bud(address(oldLpPip)) returns (uint256 kissed) {
+            assertEq(kissed, 1); // Only if a Median
+        } catch {}
+        try MedianAbstract(oldLpPip.orb1()).bud(address(oldLpPip)) returns (uint256 kissed) {
+            assertEq(kissed, 1); // Only if a Median
+        } catch {}
+
+        hevm.warp(block.timestamp + 3601);
+        oldLpPip.poke();
+        hevm.warp(block.timestamp + 3601);
+        oldLpPip.poke();
+
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        lpPip.poke();
+        hevm.warp(block.timestamp + 3601);
+        lpPip.poke();
+        spotter.poke(ilk);
+
+        assertEq(lpPip.wat(),  oldLpPip.wat());
+        assertEq(lpPip.src(),  oldLpPip.src());
+        assertEq(lpPip.orb0(), oldLpPip.orb0());
+        assertEq(lpPip.orb1(), oldLpPip.orb1());
+
```

Comment:
```suggestion

assertEq(osmMom.osms(ilk), address(lpPip));
(spotterPip,) = spotter.ilks(ilk);
assertEq(spotterPip, address(lpPip);
```

Reply:
Actually this check shows that adding the `osm` to the `osmMom` is missing. Great catch.

Reply:
Sorry I didn't say anything, it is included in `allowOSMFreeze` function

Reply:
Done